Resources/Views/App/YearX/Pages/FAQ/faq.leaf

```diff
@@ -54,6 +54,13 @@
                 </div>
             </div>
 
+            <div class="row">
+                <div class="col-xs-12 mb-4">
+                    <h3 class="h3">How do I apply for a diversity ticket?</h3>
+                    <p>If you come from a under-represented or less privileged group and would like to apply for a diversity ticket, please send us an <a href="mailto:info@serversideswift.info">email</a> with a few words about yourself and why you want to participate in the conference.</p>
```

Comment:
We might miss an `n` here in `an under-represented`. Other than that, sounds good to me 😊!

Reply:
Good catch 💪 Updated